src/change/sdp.rs

```diff
@@ -1152,17 +1211,40 @@ impl From<MediaKind> for MediaType {
 
 struct AsSdpParams<'a, 'b> {
     pub candidates: Vec<Candidate>,
-    pub creds: &'a IceCreds,
+    pub creds: IceCreds,
     pub fingerprint: &'a Fingerprint,
```

Comment:
Maybe clone Fingerprint and drop `'a` altogether? Starting to become a bit dubious :)

Reply:
Feels like a follow-up PR since it's unrelated to this change

Reply:
Sure